src/main/groovy/life/qbic/datamodel/datasets/datastructure/files/maxquant/Summary.groovy

```diff
@@ -0,0 +1,32 @@
+package life.qbic.datamodel.datasets.datastructure.files.maxquant
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+
+
+/**
+ * A specialisation of a DataFile, represents an MaxQuant Summary file
+ *
+ * @since 2.10.0
+ *
+ */
+class Summary extends DataFile{
+
+    final private static String FILE_TYPE = "pdf"
+
+    final private static String NAME_SCHEMA = $/summary_[0-9]{4}.*/$
+
+    protected Summary(String name, String relativePath) {
+        super(name, relativePath, FILE_TYPE)
+        validateName()
+    }
+
```

Comment:
Since this is a public method it would make sense to include javadocs akin to the nfcore datafiles